app/policies/user_policy.rb

```diff
@@ -0,0 +1,19 @@
+class UserPolicy < ApplicationPolicy
+  attr_reader :user, :post
+
+  def initialize(user, _post)
+    @user = user
+  end
+
+  def update?
+    user.present? || user.admin?
```

Comment:
does it mean that non admin user can do the update also?

Reply:
I changed it. For now only admin can update users.